asab/library/providers/azurestorage.py

```diff
@@ -126,7 +129,7 @@ async def list(self, path: str) -> list:
 		return items
 
 
-	async def read(self, path):
+	async def read(self, path: str) -> typing.IO:
 		assert path[:1] == '/'
 		assert '//' not in path
 		assert path[0] == '/'
```

Comment:
isn't `assert path[0] == '/'` implied by `assert path[:1] == '/'`?
why do we need both here?